targets/nrf52.json

```diff
@@ -4,6 +4,7 @@
 	"build-tags": ["nrf52", "nrf"],
 	"cflags": [
 		"--target=armv7em-none-eabi",
+		"-mfloat-abi=soft",
```

Comment:
No, it is needed to avoid compiler errors. For some reason, Clang-7 on Xenial has a problem without this flag (hits a `vfpcc` error in a CMSIS header file) - maybe a different default?
And it's fine, it only describes the ABI. It just says floats must be passed in integer registers. That's slightly less efficient but gets the job done, until either CMSIS or Clang is fixed.

Reply:
OK, works for me.